.github/workflows/ci.yml

```diff
@@ -37,7 +41,7 @@ jobs:
           # working-directory: somedir
 
           # Optional: golangci-lint command line arguments.
-          # args: --issues-exit-code=0
+          # args: --disable-all -E revive -E gosec -E errcheck -E nilerr -E govet -E gofmt -E goimports -E staticcheck
```

Comment:
Are each of these checks disabled? If so, this isn’t ideal. We desired for most checks to be enabled, understanding that some might need exceptions configured.

Reply:
That's made just for trial if it works, unfortunately nope

Reply:
Ok, and I'm just noticing that this is commented out, too, so we're good to go as-is on this line.